packages/website/components/account/paymentTable.js/paymentTable.js

```diff
@@ -79,6 +79,15 @@ const PaymentTable = ({ plans, currentPlan, setPlanSelection, setIsPaymentPlanMo
                 </div>
               </div>
             ))}
+            {currentPlan?.id === 'earlyAdopter' && (
+              <p className="early-adopter-ui-block">
+                {/* As an Early Adopter, you already get our lowest storage rate.
```

Comment:
if this isn't needed, can we remove it?

Reply:
yep, good catch. We were iterating quite fast yesterday :)